app/components/shared/inputs/ToggleInput.vue.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Prop } from 'vue-property-decorator';
+import { BaseInput } from './BaseInput';
+
+@Component({})
+export default class WBoolInput extends BaseInput<boolean, {}> {
+  @Prop() value: boolean;
+
+  handleClick(e: any) {
```

Comment:
The type you're looking for here is `KeyboardEvent`.  It's globally available, no need to import.